bukuserver/templates/bukuserver/statistic.html

```diff
@@ -223,29 +246,35 @@ <h4 class="modal-title" id="myModalLabel">Title ranking</h4>
 
 {% block tail %}
   {{ super() }}
+  <script>// sticky table headers in modals
+    $('.modal-body').each(function () {
+      $('thead', this).css({top: '-' + $.css(this, 'padding-top')});
+    });
+  </script>
   {{ buku.script('Chart.js') }}
+  {% set NO_NETLOC = '\u200B(no netloc)\u200B' %}
```

Comment:
Handling the blank name in the chart (it should be displayed, same as in table, but searching should be done against an empty string)